### PR TITLE
Project Version 2017.4

### PR DESCRIPTION
**Pull request by awjuliani**
Due to a bug in 2017.1, we are setting the default version of the UnitySDK project to 2017.4.

### CODE REVIEW

**Comment by awjuliani** [comment]
@xiaomaogy Can you update the migrating document?

File: docs/Installation.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ To install and use ML-Agents, you need install Unity, clone this repository and
 install Python with additional dependencies. Each of the subsections below
 overviews each step, in addition to a Docker set-up.
 
-## Install **Unity 2017.1** or Later
+## Install **Unity 2017.4.10f1** or Later
```
Comment (dericp) [inline docs/Installation.md]: Does it have to be 2017.4._10_ or later? Could it be any version of 2017.4?

Comment (awjuliani) [reply docs/Installation.md]: I think we should just say `2017.4`

Comment (xiaomaogy) [reply docs/Installation.md]: But we haven't tested on 2017.4's early versions, have we?

File: docs/Migrating.md [hunk 2]
```diff
@@ -9,6 +9,12 @@
   packages, `mlagents.env` and `mlagents.trainers`. `mlagents.env` can be used
   to interact directly with a Unity environment, while `mlagents.trainers`
   contains the classes for training agents.
+* The supported Unity version has changed from `2017.1 or later` to `2017.4.10f1 or later`. The lower versions might still work, but you may encounter error that looks like this:
```
Comment (dericp) [inline docs/Migrating.md]: Do we want 2017.4 here as well?

File: docs/Migrating.md [hunk 3]
```diff
@@ -9,6 +9,12 @@
...
+```
```
Comment (dericp) [inline docs/Migrating.md]: Can specify highlighting with ```console

File: docs/Migrating.md [hunk 4]
```diff
@@ -9,6 +9,12 @@
...
+Instance of CoreBrainInternal couldn't be created. The the script class needs to derive from ScriptableObject.
+UnityEngine.ScriptableObject:CreateInstance(String)
+```
+You can fix the error by removing `CoreBrain` from the CoreBrainInternal.cs:16, click on your Brain Gameobject to let the scene recompile all the changed C# scripts, then add the `CoreBrain` back. (Make sure your brain is in Internal mode, and your TensorFlowSharp plugin is imported and ENABLE_TENSORFLOW flag is set), but this fix is only valid locally and unstable.
```
Comment (dericp) [inline docs/Migrating.md]: Cap line length to 80 characters (minus code samples and things that need to be verbatim).

File: docs/Migrating.md (hunk 2 above)
Comment (dericp) [reply docs/Migrating.md]: Cap line length to 80 characters (minus code samples and things that need to be verbatim).

File: docs/Migrating.md (hunk 4 above)
Comment (unityjeffrey) [reply docs/Migrating.md]: i would consider a few changes.  In migrating.md - indicate that we are now following Unity's LTS for support going forward in the future starting with 2017.4.. .  Indicate this helps us maintain good quality and support, etc..  and that future version of ML-agents using Unity's LTS versions will be supported but only in those future versions

 I would move the 2017.1 issue into either FAQ and provide a link to it (i.e. if you are still using older versions, see here for known issues in 0.5).  We should not necessarily call it out here

Lastly, I would add a note in the main read about our moving over to the LTS supported versions of Unity.

Comment (xiaomaogy) [reply docs/Migrating.md]: @unityjeffrey 
1. What does "and that future version of ML-agents using Unity's LTS versions will be supported but only in those future versions" mean?
2. What is the "main read"？

**Pull request merged**